src/Uno.UI.Wasm/ts/Clipboard.ts

```diff
@@ -1,10 +1,42 @@
-﻿namespace Uno.Utils {
+﻿// Type declarations for Clipboard API
+// https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API
+interface Clipboard {
+	writeText(newClipText: string): Promise<void>;
+	readText(): Promise<string>;
+}
+
+interface NavigatorClipboard {
+	// Only available in a secure context.
+	readonly clipboard?: Clipboard;
+}
+
+interface Navigator extends NavigatorClipboard { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An interface declaring no members is equivalent to its supertype.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5540286)